src/game/client/components/controls.cpp

```diff
@@ -191,7 +191,7 @@ int CControls::SnapInput(int *pData)
 	if(m_pClient->m_Scoreboard.Active())
 		m_aInputData[g_Config.m_ClDummy].m_PlayerFlags |= PLAYERFLAG_SCOREBOARD;
 
-	if(m_pClient->m_Controls.m_aShowHookColl[g_Config.m_ClDummy])
+	if(m_pClient->m_Controls.m_aShowHookColl[g_Config.m_ClDummy] && (!Client()->IsSixup() || Client()->ServerCapAnyPlayerFlag()))
```

Comment:
```suggestion
	if(m_pClient->m_Controls.m_aShowHookColl[g_Config.m_ClDummy] && Client()->ServerCapAnyPlayerFlag())
```
Could you instead change the default value of `m_AnyPlayerFlag` to `!Sixup` (as a new parameter)?

https://github.com/ddnet/ddnet/blob/aaf69542b62b82605e4f691c95cbfe46eb6298a6/src/engine/client/client.cpp#L1404